source/docs/software/pathplanning/trajectory-tutorial/creating-drive-subsystem.rst

```diff
@@ -250,3 +252,5 @@ Finally, we must include one additional method - a method that allows us to set
       :lineno-start: 43
 
 It is very important to use the ``setVoltage()`` method rather than the ordinary ``set()`` method, as this will automatically compensate for battery "voltage sag" during operation.  Since our feedforward voltages are physically-meaningful (as they are based on measured identification data), this is essential to ensuring their accuracy.
+
+.. warning:: ``RamseteCommand`` itself does not internally enforce any speed or acceleration limits before providing motor voltage parameters to this method.  During initial code development, teams are strongly encouraged to apply both maximum and minimum bounds on the input variables before passing these values to ``setVoltage()``.
```

Comment:
While this is a good technique, to ensure everything is in the right phase, it can also make it so that the robot doesn't seem to follow the path, if the velocities/accelerations used to generate the trajectory are much larger then the achievable velocity/acceleration when limited.

Reply:
Would you recommend that there is an additional recommendation to ensure that the input velocity and accelerations for generation are also within the bounds of the constraint?

Reply:
@sciencewhiz

Reply:
yes